mistletoe/span_token.py

```diff
@@ -215,18 +230,21 @@ def strip(cls, string):
 
 class LineBreak(SpanToken):
     """
-    Line break token. Hard or soft.
+    Line break token: hard or soft.
     This is an inline token without children.
+
+    Attributes:
+        soft (bool): true if this is a soft line break.
     """
     repr_attributes = ("soft",)
     pattern = re.compile(r'( *|\\)\n')
     parse_inner = False
     parse_group = 0
 
     def __init__(self, match):
-        content = match.group(1)
-        self.soft = not content.startswith(('  ', '\\'))
-        self.content = ''
+        self.content = ""
```

Comment:
Yes, I think it might be there for consistency. Another option could be to use `content` instead of introducing a new attribute `marker` -- what do you think?

Reply:
Updated. This also affects the repr() output, but I don't think it's in a bad way.